### DISCUSSION TITLE
Email cliente in creazione fattura

### DISCUSSION THREAD

**Discussion by infornix** [fattureincloud/api#93] [Q&A] [answered]
Buongiorno,
non riesco a specificare l'email del cliente durante la creazione fattura.
Ho compilato il campo _entity.email_, ma nel PDF generato all'interno del portale l'email non è presente.
Inoltre provando ad inviare la fattura tramite email dal portale il campo destinatario non è precompilato.

**Comment by valmoz** [comment, marked as answer]
Buongiorno.
la nostra funzionalità non permette di inserire l'indirizzo email all'interno del PDF.

Come specificato nella guida relativa alla [creazione di fatture](https://developers.fattureincloud.it/docs/guides/invoice-creation#1%EF%B8%8F%E2%83%A3-step-one-the-customer) e nella pagina della [migrazione dalle API V1](https://developers.fattureincloud.it/docs/api-v2-migration#-documenti-emessi), inserendo un cliente nel campo entity esso non viene aggiunto in automatico nell'anagrafica clienti. Ciò significa che il metodo di creazione delle fatture non tiene conto di quei campi che, pur essendo nel modello Client, non vengono utilizzati in fase di creazione delle fatture (come nel caso dell'email).

Per memorizzare l'email è quindi necessario salvare il cliente in anagrafica utilizzando il metodo [Create Client](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/entities/clients). Ciò permetterà di precompilare il campo destinatario, ma non aggiungerà in ogni caso l'email al PDF.

In ogni caso, stiamo lavorando a riportare le funzionalità di salvataggio in anagrafica anche sulle API v2, come visibile sulla [roadmap](https://trello.com/b/uAjGCOhL/fic-api-roadmap-overview), anche se non abbiamo ancora una data definitiva di rilascio.

Resto a disposizione per ulteriori domande in merito
Mauro

**Reply by infornix** [reply]
Perfetto, grazie mille.

**Reply by damianobarbati** [reply]
@valmoz avete per caso news a riguardo?

**Reply by valmoz** [reply]
@damianobarbati se si sta riferendo al salvataggio in anagrafica, non ancora mi spiace.

**Reply by CristianBelli01k** [reply]
@valmoz any news on this? Fra un mese verranno spente le API v1 e vorrei sapere come muovermi, grazie!

**Reply by valmoz** [reply]
Buongiorno a tutti,
molto probabilmente questa feature NON verrà rilasciata prima della dismissione delle V1, quindi vi consigliamo di procedere come spiegato in precedenza.

Nel caso in cui dovessimo rilasciare prima della dismissione vi avviseremo qui sulla discussione.

Grazie
Mauro
